sql_generators/mobile_kpi_support_metrics/__init__.py

```diff
@@ -181,19 +278,28 @@ def generate(target_project, output_dir, use_cloud_function):
             )
 
             table_id = f"{target_project}.{target_dataset}.{target_name}"
+
             full_table_id = (
                 table_id + f"_{VERSION}" if target_filename == "query" else table_id
             )
 
+            product_args = {
+                "dataset": product.name,
+                "target_name": target_name,
+                "app_name": product.name,
+                "name": target_name,
+                "product_attribution_groups": product.value.attribution_groups,
```

Comment:
Would it suffice to use a shorter name suffice since this is it's called via `product.`? Also if possible, to differentiate in the naming between groups / names / fields.

Reply:
I agree that this specific aspect of the code is not the most optimal. However, I think for the purpose of making progress with getting the new profiles tables out. This is something we should do after that is done.